config.json

```diff
@@ -1080,6 +1080,21 @@
         "Pattern recognition",
         "Regular expressions"
       ]
+    },
+    {
+      "core": false,
```

Comment:
I believe this exercise must either be marked as `"core": true` or be `unlocked` by another exercise.  May I suggest the `matrix` exercise as the one to unlock this one?

Reply:
I don't remember how many exercises are marked as `core`, so I'd prefer having it as non-core. `unlocked` by `matrix` seems ok to me.

Reply:
@matthewmorgan @rchavarria  I updated the config file -  `"core": false, "unlocked_by": "matrix"`.